client/package.json

```diff
@@ -16,8 +16,10 @@
     "bootstrap": "^5.3.2",
     "react": "^18.2.0",
     "react-bootstrap": "^2.9.1",
+    "react-copy-to-clipboard": "^5.1.0",
```

Comment:
@Navdeep-av - Why do we need a package for this?

Can we use JS browser API for this?

Reply:
@xDAnkit I have uninstalled the package and using the JS browser API